config/software/unixodbc.rb

```diff
@@ -0,0 +1,28 @@
+name "unixodbc"
+default_version "2.3.4"
+
+dependency "automake"
+dependency "autoconf"
```

Comment:
I will try to build without `automake` and `autoconf` before merging this, as I think these are only build-time dependencies (and probably already in the builder images - unsure about `windows` though).